src/Analysis/Ast/Impl/Documents/RunningDocumentTable.cs

```diff
@@ -60,6 +59,10 @@ private class DocumentEntry {
         public event EventHandler<DocumentEventArgs> Closed;
         public event EventHandler<DocumentEventArgs> Removed;
 
+        public IEnumerable<IDocument> Documents {
```

Comment:
Let's make it a method then: `GetDocuments()` or something.

Reply:
Done